### ISSUE TITLE
Randomly getting “Error: token doesn’t have access to any organization"

### ISSUE THREAD

**Issue by shay-legit** [Legit-Labs/legitify#35]
### TL;DR

Randomly getting “Error: token doesn’t have access to any organization" - re-attempt of using the same token a few seconds later works just fine.

### Expected behavior

Either get results, or the actual error that happened.

### Observed behavior

The error returned is masked as “Error: token doesn’t have access to any organization" but the actual error returned from github is likely different.

### Version

v0.1.5

### On which operating system are you using legitify?

Mac OS

### Additional information

Suspecting that this “Error: token doesn’t have access to any organization" error returns when a rate limit error is returned from github.

**Comment by noamd-legit** [comment]
@shay-legit Is this still relevant? Could you provide more information?

**Comment by shay-legit** [comment]
I received this error of "token doesn’t have access to any organization" whenever my GH PAT reached the 5000/5000 rate limit. can u see how it is possible?
If not, I am OK with closing it, if it happens again I will create a new issue.

**Comment by noamd-legit** [comment]
Ok so basically this is the same as https://github.com/Legit-Labs/legitify/issues/110. We'll discuss it there.
I am closing this issue for now.
